test/warm_start.jl

```diff
@@ -0,0 +1,53 @@
+using Test
+
+@testset "Warm start" begin
+
+    using QPALM
+    using LinearAlgebra
+    using SparseArrays
+
```

Comment:
It’s better to do

```suggestion
using Random

Random.seed!(0)
```

here, to make this test less shaky.

Reply:
I do agree, thank you for the comment!